packages/build/src/core/feature_flags.js

```diff
@@ -19,4 +19,5 @@ export const DEFAULT_FEATURE_FLAGS = {
   edge_functions_cache_cli: false,
   edge_functions_produce_eszip: false,
   edge_functions_system_logger: false,
+  plugins_break_builds_with_unsupported_plugin_versions: false,
```

Comment:
NIT: can you maybe add a link to LD here and do we plan to remove this feature flag once this works successfully? So is there a follow up issue in some months to clean up?

Reply:
Done. I've set flags clean up as a task in this issue: https://github.com/netlify/pod-compute/issues/267